src/model.ts

```diff
@@ -324,6 +321,20 @@ export class Model extends EventBus<any> implements CommandDispatcher {
     return plugin;
   }
 
+  private setupCoreUiPlugin(Plugin: CorePluginConstructor) {
+    const plugin = new Plugin({ ...this.corePluginConfig, getters: this.getters });
+    for (let name of Plugin.getters) {
+      if (!(name in plugin)) {
+        throw new Error(`Invalid getter name: ${name} for plugin ${plugin.constructor}`);
+      }
+      if (name in this.getters) {
+        throw new Error(`Getter "${name}" is already defined.`);
+      }
+      this.getters[name] = plugin[name].bind(plugin);
+    }
```

Comment:
Any way to extract this loop in a method ? It's now copy/pasted 3 times in this file